alternative.platforms/amd64/packer/raspiblitz.json

```
@@ -64,9 +54,8 @@
   "variables": {
     "branch": "dev",
     "github_user": "rootzoll",
```

This file is not tested by me or the Actions, but at least the user needs to be updated:
```suggestion
    "github_user": "raspiblitz",
```